app/src/components/UserDetailsHeader.tsx

```diff
@@ -42,11 +42,11 @@ const UserDetailsHeader: React.FC = () => {
     const getFingerPrint = async () => {
       const key = await pgpLocalStorage.get();
       if (!key) return;
-
       const fingerPrint = trimFingerPrint(key.fingerPrint);
       setPgpTitle(fingerPrint);
     };
-    getFingerPrint();
+    const onFocus = navigation.addListener("focus", getFingerPrint);
+    return onFocus;
```

Comment:
Could be one line

Reply:
@brettvniekerk might be helpful to @JasminFarhi to show how?

Reply:
@JasminFarhi for return statements you don't have to predeclare a variable only to return it on the next line, you can simplify and minify your code by just returning what you would assign to that variable. In this case, you would turn lines 48 and 49 into just `return navigation.addListener("focus", getFingerPrint);`

Reply:
All done, thanks for the advice! :)